build_tools/benchmarks/export_benchmark_config.py

```diff
@@ -125,6 +125,7 @@ def _get_distinct_module_dir_paths(
 def _export_execution_handler(
     benchmark_presets: Optional[Sequence[PresetMatcher]] = None,
     target_device_names: Optional[Sequence[str]] = None,
+    shard_count: Dict[str, int] = {},
```

Comment:
Usually it's not recommended to have mutable object as default value, maybe
```suggestion
    shard_count: Optional[Dict[str, int]] = None,
```

Reply:
Add have something like `shard_count = {} if shard_count is None else shard_count` in the function body

Reply:
Changed. These Python best practice comments are very much appreciated. Thank you! :slightly_smiling_face: